tox.ini

```diff
@@ -58,8 +58,8 @@ commands_pre =
     pip freeze
 
 commands =
-    !cov: pytest --pyargs quantity {posargs} {toxinidir}/tests {toxinidir}/docs
-    cov: pytest --pyargs quantity --cov quantity --cov-report xml:coverage.xml --cov-report term-missing {posargs} {toxinidir}/tests {toxinidir}/docs
+    !cov: pytest --pyargs src {posargs} {toxinidir}/tests {toxinidir}/docs
```

Comment:
Locally, if I change this and the one below to `quantity`, things work.

Reply:
Thanks! Made the change.